core/embed/models/T3B1/boards/trezor_t3b1_revB.h

```diff
@@ -0,0 +1,63 @@
+#ifndef _TREZOR_T3B1_H
+#define _TREZOR_T3B1_H
+
+#define DISPLAY_RESX 128
+#define DISPLAY_RESY 64
+
+#define VDD_3V3 1
+
+#define USE_I2C 1
+#define USE_BUTTON 1
+#define USE_SBU 1
+#define USE_HASH_PROCESSOR 1
+#define USE_CONSUMPTION_MASK 1
+
+#include "displays/vg-2864ksweg01.h"
```

Comment:
To be compatible with `new_rendering`, we should use `#define DISPLAY_LEGACY_HEADER "displays/vg-2864ksweg01.h"` here. Please look at `trezor_r_v10.h`.

Reply:
fixed https://github.com/trezor/trezor-firmware/pull/3728/commits/a5ea7cb20e75f9cedc02d3095f1a9a6b59be21e3